interfaces/apparmor/template_vars.go

```diff
@@ -35,6 +36,11 @@ func templateVariables(info *snap.Info, securityTag string) string {
 	fmt.Fprintf(&buf, "@{SNAP_REVISION}=\"%s\"\n", info.Revision)
 	fmt.Fprintf(&buf, "@{PROFILE_DBUS}=\"%s\"\n",
 		dbus.SafePath(securityTag))
-	fmt.Fprintf(&buf, "@{INSTALL_DIR}=\"/snap\"")
+
+	if dirs.SnapMountDir == "/snap" {
+		fmt.Fprintf(&buf, "@{INSTALL_DIR}=\"/snap\"")
+	} else {
+		fmt.Fprintf(&buf, "@{INSTALL_DIR}=\"{/snap,%s}\"", dirs.SnapMountDir)
```

Comment:
You must adjust tests in the same directory. Please run `go test` to see what is broken.

Reply:
I have problem in setup those tests. I adjusted @{INSTALL_DIR} in [interfaces/apparmor/backend_test.go](https://github.com/snapcore/snapd/pull/4944/commits/21d609e8bf54dbb81f49d6999096387a4925d129#diff-28d26d7e7d320f8d0a991be41860e970R387). I don't see it being used elsewhere. If you can tell me what exactly do that would help.